packages/vaex-core/vaex/dataframe.py

```diff
@@ -5868,153 +5872,163 @@ def mangle_name(prefix, name, suffix):
             progressbar(0)
             N = len(self)
-            for i1, i2, table in self.to_arrow_table(chunk_size=chunk_size):
+            if chunk_size:
+                for i1, i2, table in self.to_arrow_table(chunk_size=chunk_size, parallel=parallel):
+                    writer.write_table(table)
+                    progressbar(i2/N)
+                progressbar(1.)
+            else:
+                table = self.to_arrow_table(chunk_size=chunk_size, parallel=parallel)
                 writer.write_table(table)
-                progressbar(i2/N)
-            progressbar(1.)
         if isinstance(path_or_writer, str):
             with pa.OSFile(path_or_writer, 'wb') as sink:
                 writer = pa.RecordBatchStreamWriter(sink, schema)
                 write(writer)
         else:
             write(path_or_writer)
 
-    def export_parquet(self, path, column_names=None, byteorder="=", shuffle=False, selection=False, progress=None, virtual=True, sort=None, ascending=True):
-        """Exports the DataFrame to a parquet file
+    @docsubst
+    def export_parquet(self, path, progress=None, parallel=True, **kwargs):
+        """Exports the DataFrame to a parquet file.
+
+        Note: This may require that all of the data fits into memory (mmapped data is an exception).
+            Use :py:`DataFrame.export_chunks` to write to multiple files in parallel.
 
-        :param DataFrameLocal df: DataFrame to export
         :param str path: path for file
-        :param lis[str] column_names: list of column names to export or None for all columns
-        :param str byteorder: = for native, < for little endian and > for big endian
-        :param bool shuffle: export rows in random order
-        :param bool selection: export selection or not
         :param progress: progress callback that gets a progress fraction as argument and should return True to continue,
                 or a default progress bar when progress=True
-        :param: bool virtual: When True, export virtual columns
-        :param str sort: expression used for sorting the output
-        :param bool ascending: sort ascending (True) or descending
+        :param bool parallel: {evaluate_parallel}
+        :param **kwargs: Extra keyword arguments to be passed on to pyarrow.parquet.write_table
         :return:
         """
-        import vaex.arrow.export
-        vaex.arrow.export.export_parquet(self, path, column_names, byteorder, shuffle, selection, progress=progress, virtual=virtual, sort=sort, ascending=ascending)
+        import pyarrow.parquet as pq
+        table = self.to_arrow_table(chunk_size=None, parallel=False)
+        pq.write_table(table, path, **kwargs)
 
-    def export_hdf5(self, path, column_names=None, byteorder="=", shuffle=False, selection=False, progress=None, virtual=True, sort=None, ascending=True):
+    @docsubst
+    def export_chunked(self, path, progress=None, chunk_size=default_chunk_size, parallel=True):
```

Comment:
This is a good idea! However I am not sure if this is a good name of this method. Maybe we should discuss this.

Reply:
export_many ? export_parallel ?